### ISSUE TITLE
Trigger Workflow receiving email 

### ISSUE THREAD

**Issue by u2mike** [aelassas/Wexflow#285]
**Describe the solution you'd like**
is there a possibility to create a workflow like this:
+ frequently checking mail account and parse received emails:
++ if defined sender and if defined subject
++ do this or that ...
thx in advance for your assistence ... any hint welcome

regards michael

**Comment by aelassas** [comment]
No, it's not possible for the moment. This would be a feature request.

**Comment by u2mike** [comment]
thx for quick response ... and with the existing "MailsReceiver" there is NO workaround possible?

**Comment by aelassas** [comment]
No, MailsReceiver just fetches emails for the moment. This task would be used to implement the feature request.

**Comment by u2mike** [comment]
ok awaiting for your feature request ... :)

to add ... thx for sharing this tool ... i use this very often 👍
